common/utils/utils.go

```diff
@@ -561,7 +561,7 @@ func GetLegacyModeSupportedDrivers() []string {
 // ReadSpannerSchema fills conv by querying Spanner infoschema treating Spanner as both the source and dest.
 func ReadSpannerSchema(ctx context.Context, conv *internal.Conv, client *sp.Client) error {
 	infoSchema := spanner.InfoSchemaImpl{Client: client, Ctx: ctx, TargetDb: conv.TargetDb}
-	err := common.ProcessSchema(conv, infoSchema)
+	err := common.ProcessSchema(conv, infoSchema, 0)
```

Comment:
Better to define a constant of 20 workers and use that instead of 0 for better readability